.github/pull_request_template.md

```diff
@@ -1,51 +1,50 @@
 <!--
-Thank you for contributing to the MariaDB Server repository!
-
-You can help us review your changes faster by filling in this template <3
-
-If you have any questions related to MariaDB or you just want to hang out and meet other community members, please join us on https://mariadb.zulipchat.com/ .
+  Thank you for contributing to the MariaDB Server repository!
+  You can help us review your changes faster by completing this template. :orange_heart:
+  If you have any questions related to MariaDB or only to meet other community members, please join us on https://mariadb.zulipchat.com/.
 -->
 
 <!--
-If you've already identified a https://jira.mariadb.org/ issue that seems to track this bug/feature, please add its number below.
+  If you've already identified a https://jira.mariadb.org/ issue that tracks this bug/feature, please add its number below.
 -->
-- [x] *The Jira issue number for this PR is: MDEV-______*
+- [x] *The Jira issue number for this PR is: [MDEV-_____](https://jira.mariadb.org/browse/MDEV-_____)*
```

Comment:
Alternatively, we could use some [custom autolinks](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources), though that’s a paid feature.